debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/BinlogReader.java

```diff
@@ -76,6 +83,7 @@
 
     private static final long INITIAL_POLL_PERIOD_IN_MILLIS = TimeUnit.SECONDS.toMillis(5);
     private static final long MAX_POLL_PERIOD_IN_MILLIS = TimeUnit.HOURS.toMillis(1);
+    private static KeyManager[] KMS = null;
```

Comment:
Is it necessary to have the variable as a static attribute? Cannot it be converted to a local variable in the method that initializes it? If yes please also rename it from uppercase.

Reply:
Yes we can, but that would entail us creating a new variable, since it is being used here

Without putting it as static we get the following error: Variable 'kms' is accessed from within inner class, needs to be final or effectively final

I will do the same and push.